src/ekat/util/ekat_array_io_mod.f90

```diff
@@ -0,0 +1,39 @@
+module array_io_mod
```

Comment:
Just checking: we don't want to name this module `ekat_array_io_mod`, do we?

Reply:
Mmm, good point. I think it's probably best to do it. The `ekat_` prefix in the module name will serve as an analogue of the C++ namespace, in the quite rare case that there already is an array_io_mod defined elsewhere. What do you think?

Reply:
I agree it's probably a good idea. Unlikely that omitting the ekat_ prefix would cause a collision, but it's a very easy thing to add.